ui/tsconfig.json

```diff
@@ -16,6 +16,7 @@
     ],
     "lib": [
       "es2016",
+      "es2018.promise",
```

Comment:
where is this used?

Reply:
it gives us `.finally` on promises - https://github.com/all-of-us/workbench/pull/2327/files#diff-86ce5d10b2dc782d34328e59e16c73cfR120